metaflow/plugins/aip/tests/flows/flow_triggering_flow.py

```diff
@@ -104,22 +102,20 @@ def end(self):
         """Trigger downstream pipeline and test triggering behaviors"""
         if self.trigger_enabled:
             logger.info("\nTesting run_kubeflow_pipeline")
-            run_id, run_uid = run_argo_workflow(
-                KUBERNETES_NAMESPACE,
-                self.template_name,
+            argo_helper = ArgoHelper(KUBERNETES_NAMESPACE)
+            run_id, run_uid = argo_helper.template_submit(
```

Comment:
for most users this could all be one line?
```
run_id, run_uid = ArgoHelper(KUBERNETES_NAMESPACE).template_submit(
```
I share this because this code will be copy-pasted and reused.

Reply:
It's two line here because the argo_helper is reused below. Let me write up a doc for how to use this function instead, so that our users are not referencing test cases.

How do you feel about namespace in each command v.s. namespace declaration in the class init once? I assume this wouldn't change customer experience much?

Reply:
I think namespace is ideally an optional parameter because we don't yet support triggering a workflowtemplate in another namespace :)

Reply:
Added.